app/src/main/java/io/gnosis/safe/ui/safe/empty/NoSafeFragment.kt

```diff
@@ -29,7 +29,7 @@ class NoSafeFragment : BaseViewBindingFragment<FragmentNoSafesBinding>() {
         super.onViewCreated(view, savedInstanceState)
 
         binding.loadSafeButton.setOnClickListener {
-            requireParentFragment().findNavController().navigate(R.id.action_to_add_safe_nav)
+            requireParentFragment().findNavController().navigate(R.id.action_to_add_safe)
```

Comment:
Can't you here the typed safe nav args here?

Reply:
Also for the other ones. Like `AddSafeFragment.globalActionId` or something?

Reply:
we don't need any arguments here.  We just navigating to the point defined by the action id. In this specific line I have renamed the action for a better readability. It had nav suffix from the times it pointed to another subgraph.

Reply:
In this case though, this no longer "refactor" proof, as this will fail in runtime instead of compile time. I agree with you, arguments are not necessary, but the destination classes that get generated are helpful in case of a refactor so we can fail in compile time instead of runtime.